### PR TITLE
Setup Travis CI (again?)

### PR DESCRIPTION
**Pull request by sauloperez**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,9 +3,5 @@ language: ruby
 rvm:
   - 2.1.4
```
Comment (andresgutgon) [inline .travis.yml]: is this needed?

File: .travis.yml [hunk 2]
```diff
@@ -3,9 +3,5 @@ language: ruby
...
 
-branches:
-  only:
-    - develop
```
Comment (andresgutgon) [inline .travis.yml]: Why remove this?

Comment (sauloperez) [inline .travis.yml]: I want to start from scratch following Travis' docs. Actually I don't remember what this means.

File: .travis.yml (hunk 1 above)
Comment (sauloperez) [inline .travis.yml]: There must be parity between all ruby versions in development, testing and production

File: .travis.yml (hunk 2 above)
Comment (andresgutgon) [inline .travis.yml]: I think this is to run travis in `develop` branch or branch that part from `develop`. With this setup `master` will not run om Travis.

File: .travis.yml (hunk 1 above)
Comment (andresgutgon) [inline .travis.yml]: We've `.ruby-version` file https://docs.travis-ci.com/user/languages/ruby/#Using-.ruby-version

File: .travis.yml (hunk 2 above)
Comment (enricostano) [inline .travis.yml]: That's correct

File: .travis.yml [hunk 3]
```diff
@@ -3,9 +3,5 @@ language: ruby
...
-
 before_script:
-  - "bundle exec rake db:setup"
+  - psql -c 'create database katuma_test;' -U postgres
```
Comment (enricostano) [inline .travis.yml]: Why not take task?

File: .travis.yml [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
 
+env:
+  - RAILS_ENV=test
+
+before_install:
+  - gem update bundler
+
 before_script:
-  - "bundle exec rake db:setup"
+  - psql -c 'create database katuma_test;' -U postgres
```
Comment (enricostano) [inline .travis.yml]: again, why this instead of rake task?

File: engines/group/spec/controllers/groups_controller_spec.rb [hunk 5]
```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
-require_relative '../../../support/shared_examples/controllers.rb'
-require_relative '../../../support/authentication.rb'
+require 'support/shared_examples/controllers.rb'
+require 'support/authentication.rb'
 
-describe Api::V1::GroupsController do
+describe Group::Api::V1::GroupsController do
```
Comment (enricostano) [inline engines/group/spec/controllers/groups_controller_spec.rb]: would make sense to keep this under `engines/group/spec/controllers/api/v1/`?

**Comment by enricostano** [comment]
:clap: @sauloperez thanks for the effort!!! :muscle:

File: engines/group/spec/controllers/groups_controller_spec.rb (hunk 5 above)
Comment (sauloperez) [inline engines/group/spec/controllers/groups_controller_spec.rb]: No idea. Any help would be welcome, cause I couldn't manage to make it work. Looks like you moved the code but not the specs. I'm still figuring out how it should be.

Comment (sauloperez) [inline engines/group/spec/controllers/groups_controller_spec.rb]: I think I got it. You moved api 1 stuff to engines without the specs

Comment (enricostano) [inline engines/group/spec/controllers/groups_controller_spec.rb]: Yes sorry :grin:

File: engines/shared/Gemfile.lock [hunk 6]
```diff
@@ -0,0 +1,128 @@
+PATH
```
Comment (enricostano) [inline engines/shared/Gemfile.lock]: we don't need `Gemfile.lock` for engines, right?

File: spec/lib/group_creation_spec.rb [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # spec/lib/group_creation_spec.rb
 require 'spec_helper'
 
-describe GroupCreation do
+describe Group::GroupCreator do
```
Comment (enricostano) [inline spec/lib/group_creation_spec.rb]: we should move all this specs under their own engine, another thing is making them green of course. Next PR.

Comment (sauloperez) [inline spec/lib/group_creation_spec.rb]: Yep, just missing that.

**Comment by sauloperez** [comment]
IMO this is ready to merge

File: .travis.yml (hunk 4 above)
Comment (enricostano) [inline .travis.yml]: ping

**Comment by sauloperez** [comment]
Ups, I totally missed it @enricostano

File: engines/group/spec/controllers/groups_controller_spec.rb [hunk 8]
```diff
@@ -1,8 +1,9 @@
...
+  routes { Group::Engine.routes }
```
Comment (sauloperez) [inline engines/group/spec/controllers/groups_controller_spec.rb]: This is THE line to tests work

**Comment by enricostano** [comment]
Superseded by https://github.com/coopdevs/katuma/pull/77

**Pull request closed without merging**